spec/web_app_spec.rb

```diff
@@ -13,6 +13,7 @@ def app
     expect(last_response.body).to include("Tic Tac Toe Game")
     expect(last_response.body).to include("Please Choose Your Game Options:")
     expect(last_response.body).to include("Board Dimensions:")
-    expect(last_response.body).to include("Player Types:")
+    expect(last_response.body).to include("THREE_BY_THREE")
```

Comment:
These assertions are coupled to constants that are outside the class (view) under test. Can you find a way to make it that this test doesn't fail when the constant(s) are changed?